sdk/AWSXRayRecorder.nuspec

```diff
@@ -15,18 +15,18 @@
     <tags>AWS Amazon cloud AWSXRay XRay</tags>
     <dependencies>
       <group targetFramework=".NETFramework4.5">
-        <dependency id="AWSXRayRecorder.Core" version="2.5.1" />
-        <dependency id="AWSXRayRecorder.Handlers.AspNet" version="2.5.1" />
-        <dependency id="AWSXRayRecorder.Handlers.AwsSdk" version="2.6.1" />
-        <dependency id="AWSXRayRecorder.Handlers.SqlServer" version="2.5.1" />
-        <dependency id="AWSXRayRecorder.Handlers.System.Net" version="2.5.1" />
+        <dependency id="AWSXRayRecorder.Core" version="2.5.2" />
+        <dependency id="AWSXRayRecorder.Handlers.AspNet" version="2.5.2" />
```

Comment:
As stated in the previous comment, don't need to increase version for `AWSXRayRecorder.Handlers.AspNet` package.

Reply:
Fixed, apologies for how many times I have done this slightly off!